jenkins-slaves/jenkins-slave-graalvm/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/openshift/origin-jenkins-agent-maven:4.2
+ARG GRAALVM_VERSION=19.3.1
+
+ADD settings.xml $HOME/.m2/settings.xml
+
+
+USER root
+RUN yum install -y gcc glibc-devel zlib-devel libstdc++-static
+RUN mkdir /opt/graalvm
+RUN curl -L https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${GRAALVM_VERSION}/graalvm-ce-java11-linux-amd64-${GRAALVM_VERSION}.tar.gz | tar --strip-components=1 -xz -C /opt/graalvm/
+RUN /opt/graalvm/bin/gu install native-image
```

Comment:
Can you please consolidate these RUN statements into one so that we only have one layer for these commands?

Reply:
I never have seen the value in consolidating the RUN commands as it reduces the ability to cache layers especially in local builds, but I have done as requested.